bddl/activity_definitions/cleaning_cups_in_living_room/problem0.bddl

```diff
@@ -20,9 +20,9 @@
         (covered wineglass.n.01_1 dust.n.01_1) 
         (covered wineglass.n.01_2 dust.n.01_1) 
         (covered wineglass.n.01_3 dust.n.01_1) 
-        (covered wineglass.n.01_1 red_wine.n.01_1)
-        (covered wineglass.n.01_2 red_wine.n.01_1)
-        (covered wineglass.n.01_3 white_wine.n.01_1)
+        (filled wineglass.n.01_1 red_wine.n.01_1)
+        (filled wineglass.n.01_2 red_wine.n.01_1)
+        (filled wineglass.n.01_3 white_wine.n.01_1)
```

Comment:
This is fine, but I do think it was better before for a task about cleaning - is there a problem with covered?

Reply:
Sampling was failing, but now with some changes Eric and I have made, covered might be fine. I can revert this

Reply:
reverted